### PR TITLE
Create blacklist.txt if not existant

### PR DESCRIPTION
**Pull request by LudovicRousseau**
Closes: https://github.com/pi-hole/pi-hole/issues/1888

**By submitting this pull request, I confirm the following:** 

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [ ] I have tested my proposed changes, and have included unit tests where possible.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**
This PR fixed issue https://github.com/pi-hole/pi-hole/issues/1888:
The log contains a lot of:
```
Jan  3 11:42:35 PiHole lighttpd[553]: grep: blacklist.txt: No such file or directory
```

**How does this PR accomplish the above?:**
The installation script now creates an empty ``${piholeDir}/blacklist.txt`` file.

A similar code was used to create an empty ``${piholeDir}/whitelist.txt``. So I guess it was the good place to also create an empty ``${piholeDir}/blacklist.txt`` file.

**What documentation changes (if any) are needed to support this PR?:**
No documentation change is needed.

### CODE REVIEW

**Comment by PromoFaux** [comment]
Hi there,

Thanks for the contribution... however one thing to pick up on, from the Contributors guide (that you have ticked to say you read 😉 ):

>- Submit Pull Requests to the **development** branch only.
>- Before Submitting your Pull Request, merge development with your new branch and fix any conflicts. (Make sure you don't break anything in development!)

I can switch that this time for you as your changes do not bring any additional commits in from `master` to `development`, however in future please base all changes on the `development` branch, and PR against it. Cheers!

File: advanced/Scripts/list.sh [hunk 1]
```diff
@@ -84,6 +84,11 @@ PoplistFile() {
     touch ${whitelist}
   fi
 
+  # Check blacklist file exists, and if not, create it
+  if [[ ! -f ${blacklist} ]]; then
```
Comment (DL6ER) [inline advanced/Scripts/list.sh]: We should probably have `${...}` here and below

Comment (PromoFaux) [reply advanced/Scripts/list.sh]: Actually, yeah, good shout. Probably on the whitelist part above, too.

@LudovicRousseau, could you please change `${blacklist}` to `"${blacklist}"` (lines 88 + 89) and `${whitelist}` to `"${whitelist}"` (lines 83 + 84)

**Comment by jacobsalmela** [comment]
@LudovicRousseau Please contact me via email for a special thank you for your contribution to our project.

**Pull request merged**